docs/README.md

```diff
@@ -60,7 +61,15 @@ According to the `minicart.v2` composition, it can be highly customizable using
 ```jsonc
 // This is the default blocks implementation for the minicart-layout
 {
+  "icon-expand#minicart-icon": {
+    "props": {
+      "size": 24
+    }
+  },
```

Comment:
I would rather move this bellow the minicart definition.

I think the top-down method is clearer. Bigger thing > to smaller things.

Reply:
I always do the opposite haha (define first and then use it). But I think your approach makes more sense

Reply:
More on this from Clean Code:
https://dzone.com/articles/the-stepdown-rule